libminifi/test/unit/StringUtilsTests.cpp

```diff
@@ -615,6 +615,16 @@ TEST_CASE("string::parseCharacter tests") {
   CHECK(string::parseCharacter("") == std::nullopt);
 }
 
+TEST_CASE("string::replaceEscapedCharacters tests") {
+  CHECK(string::replaceEscapedCharacters("a") == "a");
+  CHECK(string::replaceEscapedCharacters(R"(\n)") == "\n");
+  CHECK(string::replaceEscapedCharacters(R"(\t)") == "\t");
+  CHECK(string::replaceEscapedCharacters(R"(\r)") == "\r");
+  CHECK(string::replaceEscapedCharacters(R"(\\s)") == "\\\\s");
```

Comment:
I would add
```
CHECK(string::replaceEscapedCharacters(R"(\s)") == "\\s");
```
(or
```
CHECK(string::replaceEscapedCharacters(R"(\s)") == "s");
```
if we decide to change it)

Reply:
I've changed these based on your suggestion: https://github.com/apache/nifi-minifi-cpp/pull/1753/commits/3b6a8c40b0f991513b71e3c2a53bef7b6a088acc